app/models/concerns/deliverable.rb

```diff
@@ -3,7 +3,7 @@ module Deliverable
   extend Suppressible
 
   included do
-    after_create_commit :deliver_later, unless: -> { Deliverable.suppressed? }
+    after_create_commit :deliver_later, if: -> { deliver_after_creation? }
```

Comment:
this should also work (i haven't tested it)

```suggestion
    after_create_commit :deliver_later, if: :deliver_after_creation?
```